src/compute-pr-actions.ts

```diff
@@ -70,6 +70,16 @@ export function process(info: PrInfo): Actions {
         status: createWelcomeComment(info)
     });
 
+    // Ping reviewers
+    if (!info.anyPackageIsNew && info.owners.length) {

```

Comment:
Why `!info.anyPackageIsNew`? You could have a PR that adds a new package but also touches packages with owners that should be pinged, right?

Reply:
Yeah, that's reasonable - I think the length should be enough in that case then